nengo_gui/static/components/raster.js

```diff
@@ -119,3 +127,57 @@ Nengo.Raster.prototype.reset = function(event) {
     this.data_store.reset();
     this.schedule_update();
 }
+
+Nengo.Raster.prototype.generate_menu = function() {
+    var self = this;
+    var items = [];
+    items.push(['Set # neurons...', function() {self.set_neuron_count();}]);
+
+    return $.merge(items, Nengo.Component.prototype.generate_menu.call(this));
+};
+
+Nengo.Raster.prototype.set_neuron_count = function() {
+    var count = this.n_neurons;
+    var self = this;
+    Nengo.modal.title('Set number of neurons...');
+    Nengo.modal.single_input_body(count, 'Number of neurons');
+    Nengo.modal.footer('ok_cancel', function(e) {
+        var new_count = $('#singleInput').val();
+        var modal = $('#myModalForm').data('bs.validator');
+        modal.validate();
+        if (modal.hasErrors() || modal.isIncomplete()) {
+            return;
+        }
+        if (new_count !== null) {
+            new_count = parseInt(new_count);
+            self.set_n_neurons(new_count);
+            self.axes2d.fit_ticks(self);
+        }
+        $('#OK').attr('data-dismiss', 'modal');
+    });
+    var $form = $('#myModalForm').validator({
+        custom: {
+            my_validator: function($item) {
+                var num = $item.val();
+                var valid = false;
+                if ($.isNumeric(num)) {
+                    num = Number(num);
+                    if (num >= 0 && Number.isInteger(num)) {
+                        valid = true;
+                    }
+                }
+                return valid;
+            }
+        },
+    });
+
+    $('#singleInput').attr('data-error', 'Input should be a positive integer');
+
+    Nengo.modal.show();
+    $('#OK').on('click', function () {
```

Comment:
No spaces after `function`. Have you run this through `jscs`? Do you want me to do it for you?